include/fizzy/fizzy.h

```diff
@@ -157,6 +157,15 @@ const FizzyModule* fizzy_parse(const uint8_t* wasm_binary, size_t wasm_binary_si
 /// If passed pointer is NULL, has no effect.
 void fizzy_free_module(const FizzyModule* module);
 
+/// Make a copy of a module.
+///
+/// @param  module          Pointer to module. Cannot be NULL.
+/// @returns                Pointer to a newly allocated module, identical to @a module.
+///
+/// @note Creating a copy is required if more than single instance of a module is required, because
```

Comment:
```suggestion
/// @note Creating a copy is required if more than single instance of a module is needed, because
```
or
```suggestion
/// @note Creating a copy is needed if more than single instance of a module is required, because
```